frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -162,6 +208,55 @@ export const TextField = ({
     );
   }
 
+  if (isAutoCompleteable) {
+    const [autoCompleteOptions, setAutoCompleteOptions] = React.useState([]);
+    const autoCompleteDebounce = React.useRef(
+      _.debounce(value => {
+        autocompleteCallback(value)
+          .then(data => {
+            setAutoCompleteOptions(data.results);
+          })
+          .catch(err => {
+            helpText = err;
+          });
+      }, 500)
+    ).current;
+
+    // TODO (mcutalo): support option.label in the renderOption
+    return (
+      <Autocomplete
+        freeSolo
+        size="small"
+        options={autoCompleteOptions}
+        PopperComponent={renderPopper}
```

Comment:
```suggestion
        PopperComponent={Popper}
```

Reply:
ah wow i didnt know it would just pass down the props for you nice.